lang_tests/simple.c

```diff
@@ -1,6 +1,48 @@
 // Compiler: 
 // Runtime:
+//    stdout: int args: -3, -1, -47, -5, 135, -6, -4004, -1, 0, 1, -1, 44, 1
+//            double args: 5.47, 13.79, 10.558, 1, -1, 45.5104, -27.60672, 1.5, 0.8
+//            float args: 8, 10, 9, 6, -1, 16, -20, 4, 2
 int main() {
```

Comment:
Huh, what happened to the `printf` tests? I think we should leave them intact and then create a new test file for binary operations. And while we are here, we should probably split the binops up into multiple test files as well, e.g. one for addition, subtraction, etc.